actioncable/Rakefile

```diff
@@ -25,12 +25,7 @@ namespace :test do
   end
 
   task :integration do
-    require "blade"
-    if ENV["CI"]
-      Blade.start(interface: :ci)
-    else
-      Blade.start(interface: :runner)
-    end
+    system("yarn install && yarn test") || raise("Failures")
```

Comment:
Not loving running `yarn install` from the test command. Let's figure out the Right Way™ and move it to `.travis.yml`. Here-ish?https://github.com/rails/rails/blob/14e49c2e28e2be4d97d132e99dd92ae9c4ddb4a9/.travis.yml#L40-L44 Perhaps @rafaelfranca can offer a little guidance. 😘

Reply:
Try removing `yarn install` and adding a line to `.travis.yml` similar to https://github.com/rails/rails/blob/14e49c2e28e2be4d97d132e99dd92ae9c4ddb4a9/.travis.yml#L42

Reply:
Moved `yarn install` to `.travis.yml` in https://github.com/rmacklin/rails/commit/bdc448a959525d10b72be0af8e4114c484cf2228